crates/sui-types/src/messages.rs

```diff
@@ -2026,6 +2028,16 @@ impl ConsensusTransaction {
         }
     }
 
+    pub fn new_checkpoint_signature_message(data: SignedCheckpointData) -> Self {
+        let mut hasher = DefaultHasher::new();
+        data.hash(&mut hasher);
```

Comment:
It's probably not necessary to hash the entire thing. I think you just need to hash the signature since it's a also commit to the hash?

Reply:
Makes sense, updated